Source/Core/Core/ConfigManager.cpp

```diff
@@ -992,14 +992,7 @@ bool SConfig::AutoSetup(EBootBS2 _BootBS2)
           DiscIO::CNANDContentManager::Access().GetNANDLoader(m_strFilename);
       const IOS::ES::TMDReader& tmd = content_loader.GetTMD();
 
-      if (content_loader.GetContentByIndex(tmd.GetBootIndex()) == nullptr)
-      {
-        // WAD is valid yet cannot be booted. Install instead.
-        u64 installed = DiscIO::CNANDContentManager::Access().Install_WiiWAD(m_strFilename);
-        if (installed)
-          SuccessAlertT("The WAD has been installed successfully");
-        return false;  // do not boot
-      }
+      _assert_(content_loader.GetContentByIndex(tmd.GetBootIndex()) != nullptr);
```

Comment:
We should return false here (in addition to displaying a message) so that Dolphin won't try to boot garbage if the user for instance tries to launch a DLC WAD using File > Open.